go/logictest/resultparser.go

```diff
@@ -120,17 +145,49 @@ func parseLogEntry(scanner *parser.LineScanner) (*ResultLogEntry, error) {
 			panic(fmt.Sprintf("Failed to parse line number on line %v", scanner.LineNum))
 		}
 
-		switch entry.Result {
-		case NotOk:
-			eoq := strings.Index(line[colonIdx2+1:], "not ok: ") + colonIdx2 + 1
-			entry.Query = line[colonIdx2+2 : eoq-1]
-			entry.ErrorMessage = line[eoq+len("not ok: "):]
-		case Ok:
-			eoq := strings.Index(line[colonIdx2+1:], "ok") + colonIdx2 + 1
-			entry.Query = line[colonIdx2+2 : eoq-1]
-		case Skipped:
-			eoq := strings.Index(line[colonIdx2+1:], "skipped") + colonIdx2 + 1
-			entry.Query = line[colonIdx2+2 : eoq-1]
+
+		if lineHasDurations {
```

Comment:
Yeah this is way too much work. Just put the duration immediately after the timestamp without any punctuation, makes the rest of the parsing unchanged.